.github/workflows/validate-swagger.yml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
is this necessary?

Reply:
This will cause it to run after it's been merged similar to the wallet repo - not necessary, but consistent with other workflows. The workflow_dispatch is nice so that the tests can be run on a branch without creating a PR.

Reply:
Doesn't the push trigger cause that?

Reply:
If you want to run w/o creating a PR, I'd rather be able to run locally w/o creating a remote branch.

Reply:
+1, I'd like to be able to run this locally before pushing. If this were a Valora-only repo, I'd vote to have it run as part of a pre-commit hook (rather not be too opinionated on a non-Valora, 'documentation-only' repo though).

Reply:
Locally you can use the [Swagger Editor](https://github.com/swagger-api/swagger-editor) or [Web Swagger Editor](https://editor.swagger.io/). I can add this to the `README.md`.

Reply:
@silasbw Would prefer it on pull request instead of push so that the pull requests would need to pass checks prior to merging.

Reply:
sorry, running locally using a CLI tool.  e.g.,

```
yarn validate
```

Reply:
...this also makes it easy to use the same check locally as in CI (vs two different tools)

Reply:
Yeah, that's what I was thinking as well; would love to be able to run validation on the spec via CLI

Reply:
@silasbw @jophish swapped out the remote [Swagger Editor](https://github.com/swagger-api/swagger-editor) for [@apidevtools/swagger-cli](https://www.npmjs.com/package/@apidevtools/swagger-cli). This should allow users of this repo to run the same validation locally with `yarn validate` as would be run in the CI checks.